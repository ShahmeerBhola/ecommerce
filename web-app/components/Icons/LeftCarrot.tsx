import React from 'react';
import config from '../../config';

type Props = {
  onClick: () => void;
};

const LeftCarrot: React.FunctionComponent<Props> = ({ onClick }) => {
  const red100 = config.colors['red-100'];
  return (
    <svg
      className="cursor-pointer"
      onClick={onClick}
      width="19"
      height="31"
      viewBox="0 0 19 31"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path d="M1 15.5L16.1522 30L18 28.2317L4.69565 15.5L18 2.76829L16.1522 1L1 15.5Z" fill={red100} stroke={red100} />
    </svg>
  );
};

export default LeftCarrot;
