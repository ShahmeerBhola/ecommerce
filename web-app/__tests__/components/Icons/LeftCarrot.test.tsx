import React from 'react';
import renderer from 'react-test-renderer';
import LeftCarrot from '../../../components/Icons/LeftCarrot';

describe('LeftCarrot component', () => {
  test('it renders properly', () => {
    const tree = renderer.create(<LeftCarrot onClick={(): void => console.log('hello')} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
