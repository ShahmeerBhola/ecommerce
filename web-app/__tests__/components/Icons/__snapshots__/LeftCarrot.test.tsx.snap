// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftCarrot component it renders properly 1`] = `
<svg
  className="cursor-pointer"
  fill="none"
  height="31"
  onClick={[Function]}
  viewBox="0 0 19 31"
  width="19"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M1 15.5L16.1522 30L18 28.2317L4.69565 15.5L18 2.76829L16.1522 1L1 15.5Z"
    fill="#F23939"
    stroke="#F23939"
  />
</svg>
`;
